content/about/index.md

```diff
@@ -115,5 +115,8 @@ of the following members:
 - [Valentin Sulzer](https://www.linkedin.com/in/valentinsulzer/) - Ionworks
 - [Martin Robinson](https://www.sabsr3.ox.ac.uk/people/dr-martin-robinson) - University of Oxford
 
-For more details, see the [PyBaMM Governance](/governance) page. The teams of maintainers and maintainer trainees are
-listed on the [Teams](/teams) page.
+For more details, see the [PyBaMM Governance](/governance) page.
+
+## Current PyBaMM Team
+
+The teams of maintainers and maintainer trainees are listed on the [Teams](/teams) page.
```

Comment:
It took me around an hour to find the teams page on the website because it was buried under the Steering Council section 😬

Reply:
Haha, yes, while the accessibility was fixed, we definitely need to improve the navigation. We should add either a sitemap or ensure that every main/heading page is placed in the footer of the website (and add more footer columns as necessary).

Reply:
I realise the teams page was also in the footer btw lol